test/specs/absolute-root/absolute-root.spec.js

```diff
@@ -0,0 +1,38 @@
+"use strict";
+
+const { expect } = require("chai");
+const $RefParser = require("../../..");
+const path = require("../../utils/path");
+const url = require("../../../lib/util/url");
+
+describe("When executed in the context of root directory", () => {
+  const { cwd } = url;
+  const { cwd: processCwd } = process;
+
+  beforeEach(() => {
+    url.cwd = function () {
+      try {
+        process.cwd = () => "/";
+        return cwd.apply(null, arguments);
+      }
+      finally {
+        process.cwd = processCwd;
+      }
+    };
+  });
+
+  afterEach(() => {
+    url.cwd = cwd;
+    process.cwd = processCwd; // already restored at line 19, but just in case
+  });
+
+  it("should resolve successfully a $ref pointing at absolute location", async () => {
+    let parser = new $RefParser();
+    await parser.parse(path.abs("specs/absolute-root/empty-object.json"));
+    expect(parser.$refs._$refs).to.have.keys([
```

Comment:
Instead of testing against the internal `_$refs` property, you should use the `parser.$refs.paths()` method.  Or, better yet, use the `helper.testResolve()` utility.  See "test/specs/external/external.spec.js" for example usage.